ea_restaurant_data_loader/config/dev.exs

```diff
@@ -55,3 +55,5 @@ config :phoenix, :stacktrace_depth, 20
 
 # Initialize plugs at runtime for faster development compilation
 config :phoenix, :plug_init_mode, :runtime
+
+config :joken, default_signer: "secret"
```

Comment:
why did you add another `config` instead of using the existing one above?

Reply:
This file have many config statements, as joken is not related with the configs above i added separately.

Reply:
Makes sense to me